user_guide_src/source/libraries/encryption/013.php

```diff
@@ -1,9 +1,8 @@
 <?php
 
 use Config\Encryption;
-use Config\Services;
 
-$config         = new Encryption();
+$config         = config(Encryption::class);
 $config->driver = 'OpenSSL';
```

Comment:
This change is not good. Because you change the value of the shared `Config\Encryption` instance.
Do not use `config()` in this case.